types/src/account_state.rs

```diff
@@ -337,6 +339,38 @@ impl fmt::Debug for AccountState {
     }
 }
 
+impl TryFrom<&StateStoreValue> for AccountState {
+    type Error = Error;
+
+    fn try_from(state_store_value: &StateStoreValue) -> Result<Self> {
+        AccountState::try_from(&state_store_value.bytes)
+    }
+}
+
+impl TryFrom<&mut StateStoreValue> for AccountState {
+    type Error = Error;
+
+    fn try_from(state_store_value: &mut StateStoreValue) -> Result<Self> {
+        AccountState::try_from(&state_store_value.bytes)
+    }
+}
```

Comment:
Is this necessary?

Reply:
Yes there are some tests still generating arbitrary account state, so this is needed.